tests/auth.test.ts

```diff
@@ -1,16 +1,17 @@
 import Client from "@/sdk-js/dist";
-import { WorkflowStatus } from "@/types/dist";
+import { GetKeyRequestSignature, WorkflowStatus } from "@/types/dist";
 import dotenv from "dotenv";
 import path from "path";
 import {
   getAddress,
-  generateSignature, generateAuthPayloadWithApiKey,
+  generateSignature,
+  generateAuthPayloadWithApiKey,
   requireEnvVar,
   queueForRemoval,
   removeCreatedWorkflows,
   compareResults,
 } from "./utils";
-import { EXPIRED_AT, FACTORY_ADDRESS, WorkflowTemplate } from "./templates";
```

Comment:
Removed this EXPIRED_AT and used a `EXPIRATION_DURATION_MS` instead, and since it’s always inside the generateKeyRequestMessage() function I moved this constant to inside the function.